includes/class-wc-stripe-helper.php

```diff
@@ -439,73 +439,33 @@ public static function get_settings( $method = null, $setting = null ) {
 	 * List of legacy payment method classes.
 	 *
 	 * @return array
+	 *
+	 * @deprecated 10.1.0 This method will be removed in future versions.
```

Comment:
Nit: Another version fix.
```suggestion
	 * @deprecated 10.2.0 This method will be removed in future versions.
```